docusaurus.config.js

```diff
@@ -7,7 +7,7 @@ const a11yEmoji = require('@fec/remark-a11y-emoji');
 const config = {
   title: 'Resoto by Some Engineering Inc.',
   tagline:
-    'Find leaky resources, manage quota limits, detect drift, and clean up!',
+    'Automate tedious infrastructure tasks, remarkably fast!',
```

Comment:
```suggestion
    'Effortlessly automate tedious infrastructure tasks!',
```
I'm not sure if "remarkably fast" is accurate, and fast isn't always good.

Reply:
It's the tag line we're using right now. Can change it later.